sources/networkmanager/transit_gateway_peering.go

```diff
@@ -0,0 +1,133 @@
+package networkmanager
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/networkmanager"
+	"github.com/aws/aws-sdk-go-v2/service/networkmanager/types"
+	"github.com/overmindtech/aws-source/sources"
+	"github.com/overmindtech/sdp-go"
+)
+
+func getTransitGatewayPeeringGetFunc(ctx context.Context, client *networkmanager.Client, _, query string) (*types.TransitGatewayPeering, error) {
+	out, err := client.GetTransitGatewayPeering(ctx, &networkmanager.GetTransitGatewayPeeringInput{
+		PeeringId: &query,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return out.TransitGatewayPeering, nil
+}
+
+func transitGatewayPeeringItemMapper(scope string, awsItem *types.TransitGatewayPeering) (*sdp.Item, error) {
+	attributes, err := sources.ToAttributesCase(awsItem)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// The uniqueAttributeValue for this is a nested value of peeringId:
+	if awsItem != nil && awsItem.Peering != nil {
+		attributes.Set("peeringId", *awsItem.Peering.PeeringId)
+	}
+
+	item := sdp.Item{
+		Type:            "networkmanager-transit-gateway-peering",
+		UniqueAttribute: "peeringId",
+		Attributes:      attributes,
+		Scope:           scope,
+	}
+
+	if awsItem.Peering != nil {
+		if awsItem.Peering.CoreNetworkId != nil {
+			item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{
+				Query: &sdp.Query{
+					// +overmind:link networkmanager-core-network
+					// Search for core network
+					Type:   "networkmanager-core-network",
+					Method: sdp.QueryMethod_GET,
+					Query:  *awsItem.Peering.CoreNetworkId,
+					Scope:  scope,
+				},
+				BlastPropagation: &sdp.BlastPropagation{
+					In:  true,
+					Out: true,
+				},
+			})
+		}
+
+		switch awsItem.Peering.State {
+		case types.PeeringStateCreating:
+			item.Health = sdp.Health_HEALTH_PENDING.Enum()
+		case types.PeeringStateAvailable:
+			item.Health = sdp.Health_HEALTH_OK.Enum()
+		case types.PeeringStateDeleting:
+			item.Health = sdp.Health_HEALTH_PENDING.Enum()
+		case types.PeeringStateFailed:
+			item.Health = sdp.Health_HEALTH_ERROR.Enum()
+		}
+	}
+	// TODO: add support for ec2-transit-gateway-peering-attachment
+	if awsItem.TransitGatewayPeeringAttachmentId != nil {
+		item.LinkedItemQueries = append(item.LinkedItemQueries, &sdp.LinkedItemQuery{
+			Query: &sdp.Query{
+				// +overmind:link ec2-transit-gateway-peering-attachment
+				Type:   "ec2-transit-gateway-peering-attachment",
+				Method: sdp.QueryMethod_SEARCH,
```

Comment:
```suggestion
				Method: sdp.QueryMethod_GET,
```

This doesn't exist yet, but by convention since we're querying by ID this is more likely to be GET